src/appengine/handlers/cron/fuzzer_coverage.py

```diff
@@ -40,10 +41,15 @@ def _basename(gcs_path):
 def _read_json(url):
   """Returns a JSON obejct loaded from the given GCS url."""
   data = storage.read_data(url)
-  if not data:
-    return None
 
-  return json.loads(data)
+  result = None
+  try:
+    result = json.loads(data)
+  except Exception as e:
```

Comment:
usually prefer specific exception - JSONDecodeError here ?

Reply:
It can be `JSONDecodeError` or `TypeError` at least, maybe something else. Do you think it makes sense to enumerate all of them?

Reply:
and @mbarbella-chromium is right, in Python2 `ValueError` is also a possibility. I frankly don't think it's reasonable to distinguish given that there is a single function call inside `try` block, but would love to have an agreement between us

Reply:
ok sure, just generic is fine as-is.